### PR TITLE
Settings/Log: Print advanced settings warning on the log

### PR DESCRIPTION
**Pull request by kamfretoz**
### Description of Changes

This PR adds a log entry that warns the user through the log if they have advanced options enabled.
Preview:
image

### Rationale behind Changes

Helps with user support in the event a user has messed around with the advanced settings.

### Suggested Testing Steps

See if the warning shows up correctly if the advanced settings are enabled. Also checks when its off too, or when the value is absent from the ini file.

### CODE REVIEW

**Review by JordanTheToaster — APPROVED** [review]

**Review by Mrlinkwii — APPROVED** [review]
image
looks good to me

File: pcsx2/VMManager.cpp [hunk 1]
```diff
@@ -415,6 +415,11 @@ bool VMManager::Internal::CPUThreadInitialize()
 	if (EmuConfig.EnableDiscordPresence)
 		InitializeDiscordPresence();
 
+	// Check for advanced settings status and warn the user if its enabled
+	bool advanced_settings = Host::GetBaseBoolSettingValue("UI", "ShowAdvancedSettings", false);
+	if (advanced_settings)
```
Comment (F0bes) [inline pcsx2/VMManager.cpp:420]:
```suggestion
	if (Host::GetBaseBoolSettingValue("UI", "ShowAdvancedSettings", false))
```
Or make it const.

**Review by F0bes — CHANGES_REQUESTED** [review]

File: pcsx2/VMManager.cpp (hunk 1 above)
Comment (kamfretoz) [reply pcsx2/VMManager.cpp]: Done.

**Comment by kamfretoz** [comment]
Also adjusted the log formatting a bit to be more consistent.

Before:
image

After:
image

**Review by Mrlinkwii — APPROVED** [review]

File: pcsx2/R3000AInterpreter.cpp [hunk 2]
```diff
@@ -237,7 +237,7 @@ static void doBranch(s32 tar) {
 	// This detects when SYSMEM is called and clears the modules then
 	if(tar == 0x890)
 	{
-		DevCon.WriteLn(Color_Gray, "[R3000 Debugger] Branch to 0x890 (SYSMEM). Clearing modules.");
+		DevCon.WriteLn(Color_Gray, ":R3000 Debugger: Branch to 0x890 (SYSMEM). Clearing modules.");
```
Comment (F0bes) [inline pcsx2/R3000AInterpreter.cpp:240]: You've added a colon to the start of this string

**Review by F0bes — CHANGES_REQUESTED** [review]

**Review by F0bes — APPROVED** [review]
LGTM

**Pull request merged**